build/build-image/cross/VERSION

```diff
@@ -1 +1 @@
-v1.27.0-go1.20.7-bullseye.0
```

Comment:
interesting, I would have expected this to already be at 1.28.0-go1.20.7...

Reply:
fixed